backend/medtagger/api/auth/business.py

```diff
@@ -0,0 +1,45 @@
+"""Module responsible for business logic in all Auth endpoint."""
+from flask_security import login_user, logout_user
+from werkzeug.security import generate_password_hash, check_password_hash
+
+from medtagger.api import InvalidArgumentsException
+from medtagger.database.models import User
+from medtagger.repositories.roles import RolesRepository
+from medtagger.repositories.users import UsersRepository
+
+
+def create_user(email: str, password: str, first_name: str, last_name: str) -> int:
+    """Create user with the given user information. Password is being hashed.
+
+    :return: id of the new user
+    """
+    user = UsersRepository.get_user_by_email(email)
+    if user is not None:
+        raise InvalidArgumentsException('User with this email already exist')
+    password_hash = generate_password_hash(password)
+    new_user = User(email, password_hash, first_name, last_name)
+    role = RolesRepository.get_role_with_name('volunteer')
```

Comment:
Please use Enum for available Roles instead of hardcoded string.
...or create an issue if it's something that requires more work.

Reply:
I will create issue with that. It requires some work and it is better not to add everything in on PR.

Reply:
https://github.com/jpowie01/MedTagger/issues/69